### PR TITLE
Generic sql - docker image changed

### PR DESCRIPTION
**Pull request by ohaim1008**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/24218

## Description
Fixed an issue where connecting to mssql db using encrypted connection failed.

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation 

reference: https://github.com/demisto/dockerfiles/pull/1815

### CODE REVIEW

File: Packs/GenericSQL/pack_metadata.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
     "description": "Connecting ang executing sql queries in 4 Databases: MYSQL, PostgreSQL, Microsoft SQL Server and Oracle",
     "support": "xsoar",
     "serverMinVersion": "5.0.0",
-    "currentVersion": "1.0.0",
+    "currentVersion": null,
```
Comment (bakatzir) [inline Packs/GenericSQL/pack_metadata.json:6]:
```suggestion
    "currentVersion": 1.0.1,
```

File: Packs/GenericSQL/ReleaseNotes/2_0_0.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
```
Comment (bakatzir) [inline Packs/GenericSQL/ReleaseNotes/2_0_0.md:1]: rm this file please

File: Packs/GenericSQL/ReleaseNotes/1_0_1.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+- __Generic SQL__
+%%UPDATE_RN%%
```
Comment (bakatzir) [inline Packs/GenericSQL/ReleaseNotes/1_0_1.md:4]:
```suggestion
Fixed an issue where connecting to mssql db using encrypted connection failed.
```

File: Packs/GenericSQL/TestPlaybooks/generic-sql-mssql-encrypted-connection.yml [hunk 4]
```diff
@@ -0,0 +1,208 @@
+id: 8fe09eba-8482-41af-845f-218474e4d079
+version: 10
```
Comment (bakatzir) [inline Packs/GenericSQL/TestPlaybooks/generic-sql-mssql-encrypted-connection.yml:2]:
```suggestion
version: -1
```

File: Packs/GenericSQL/TestPlaybooks/generic-sql-mssql-encrypted-connection.yml [hunk 5]
```diff
@@ -0,0 +1,208 @@
+id: 8fe09eba-8482-41af-845f-218474e4d079
```
Comment (bakatzir) [inline Packs/GenericSQL/TestPlaybooks/generic-sql-mssql-encrypted-connection.yml:1]:
```suggestion
id: generic-sql-mssql-encrypted-connection
```

File: Packs/GenericSQL/pack_metadata.json [hunk 6]
```diff
@@ -3,7 +3,7 @@
     "description": "Connecting ang executing sql queries in 4 Databases: MYSQL, PostgreSQL, Microsoft SQL Server and Oracle",
     "support": "xsoar",
     "serverMinVersion": "5.0.0",
-    "currentVersion": "1.0.0",
+    "currentVersion": "1.2.1",
```
Comment (bakatzir) [inline Packs/GenericSQL/pack_metadata.json:6]:
```suggestion
    "currentVersion": "1.0.1",
```

**Comment by ohaim1008** [comment]
@kirbles19 Hi Alex,
Please review the release notes in those files:

https://github.com/demisto/content/blob/82f5e1711a651270c7286370adddf3c6a31ce6a5/Packs/GenericSQL/Integrations/GenericSQL/CHANGELOG.md
https://github.com/demisto/content/blob/82f5e1711a651270c7286370adddf3c6a31ce6a5/Packs/GenericSQL/ReleaseNotes/1_0_1.md

thanks!

**Comment by kirbles19** [comment]
@ohaim1008 Done.

**Comment by ohaim1008** [comment]
moved to https://github.com/demisto/content/pull/7174

**Pull request closed without merging**